sections/errorhandling/testingerrorflows.md

```diff
@@ -6,20 +6,37 @@ Testing ‘happy’ paths is no better than testing failures. Good testing code
 
 ### Code example: ensuring the right exception is thrown using Mocha & Chai
 
-```javascript
+<details>
+<summary><strong>Javascript</strong></summary>
+<pre lang="javascript">
 describe("Facebook chat", () => {
   it("Notifies on new chat message", () => {
     var chatService = new chatService();

```

Comment:
We could switch the `var` for a `const` and only use one example here instead of two?

Reply:
Also, but that might not be the place to discuss this, mocha state that we should avoid using arrow functions for tests: https://mochajs.org/#arrow-functions. So should we switch to `function` instead? Or maybe that's another PR?

Reply:
I think we _could_ definitely swap out the arrow functions in places where we have to use the Mocha context as stated in the docs, but for the rest it should definitely be fine (for now). If you'd like to swap those out this would be awesome though!